src/services/createListService.ts

```diff
@@ -0,0 +1,14 @@
+import axios from "axios";
+import { IListData } from "store/types";
+
+export const createListService = async (listData: IListData) => {
+  const res = await axios.post(
+    `https://young-citadel-44598.herokuapp.com/categories/`,
```

Comment:
https://young-citadel-44598.herokuapp.com - export from another place and use this for all requests